api/.env.example

```diff
@@ -0,0 +1 @@
+SQL_CONNECTION_STRING='mysql://dev:dev@127.0.0.1:3306/timetrackdev'
```

Comment:
Make sure you add a `.env` file with this in it. It's how you will connect locally now